### ISSUE TITLE
Image support (AVIF)

### ISSUE THREAD

**Issue by Pyroglyph** [alexheretic/ab-av1#56]
[AVIF](https://en.wikipedia.org/wiki/AVIF) is to images as AV1 is to video: the future.

It would be useful to be able to auto-encode images as well as video with a CLI command such as the following:

```sh
ab-av1 auto-encode -i "image.jpg"
```

Currently, this results in `Error: ffprobe missing video duration` as ab-av1 appears to assume the input is always a video. This also happens when passing an image to `ab-av1 encode`.

I know the CRF searching process also works with images as I can do it manually with the following commands:

```sh
ffmpeg -i "image.jpg" -crf 32 "image.crf32.avif"
ab-av1 vmaf --reference "image.jpg" --distorted "image.crf32.avif"
```

Then, based on the output VMAF score, the given CRF can be adjusted to achieve the desired quality/size, exactly like `ab-av1 auto-encode` currently does for videos.

**Comment by alexheretic** [comment]
I agree, this would be a nice new ability for this tool!

**Comment by alexheretic** [comment]
Closed by #57
